test/logger.js

```diff
@@ -67,6 +71,15 @@ describe('Test all Levels', function() {
             done();
         }, configs.FLUSH_INTERVAL + 200);
     });
+    it('Debug Function with callback', function(done) {
```

Comment:
I've written this in other test diffs but the `it` should describe what you expect the test to do. So something like `it('debug() succeeds with no errors')`.